types/releases.ts

```diff
@@ -20,8 +20,18 @@ export interface NodeReleaseData {
     | 'Current'
     | 'End-of-life'
     | 'Pending';
-  initialRelease: string;
-  ltsStart: string | null;
-  maintenanceStart: string | null;
+  currentStart: string;
+  ltsStart?: string;
+  maintenanceStart?: string;
   endOfLife: string;
+  npm: string;
+  v8: string;
+  releaseDate: string;
+  modules: string;
+}
+
+export interface NodeReleasesDataContext {
```

Comment:
The type of context should probably reside directly on the ContextProvider :) as it should pretty much only be used there. And it doesn't even need to be exported.